BE/src/main/java/com/codesquad/issuetracker/auth/request/AccessTokenRequest.java

```diff
@@ -0,0 +1,36 @@
+package com.codesquad.issuetracker.auth.request;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class AccessTokenRequest {
+
+    @JsonProperty("client_id")
+    private final String client_id;
+
+    @JsonProperty("client_secret")
+    private final String client_secret;
```

Comment:
네이밍에 있어서 신경 써주시면 좋을 것 같습니다.
```suggestion
    @JsonProperty("client_id")
    private final String clientId;

    @JsonProperty("client_secret")
    private final String clientSecret;
```